index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title>Darya Startsava CV</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="alignment-all">
+        <header class="color">
+            <img src="assets/photo.jpg" alt="My photo" class="photo" width="190">
+            <h1>DARYA STARTSAVA</h1>
+            <h2>Front-End Developer</h2>
+            <p>Location: Krakow, Poland</p>
+            <section>
+                <h3>Contact</h3>
+                <ul>
+                    <li>Discord: darya-startsava#4511</li>
+                    <li>Email: daryastartsava@gmail.com</li>
+                </ul>
+            </section>
+        </header>
+        <main>
+            <section>
+                <h3>About me</h3>
+                <p>Hi, I'm Darya. I am a self-taught front-end developer.
+                    I practice writing clean and elegant code.</p>
```

Comment:
reformat to make it more readable:
```suggestion
                <p>
                     Hi, I'm Darya. I am a self-taught front-end developer.
                     I practice writing clean and elegant code.
                </p>
```